packages/sdk/test/integration/gc_test.ts

```
@@ -154,51 +233,120 @@
-    // (1, 0) -> (1, 1): syncedseqs:(0, 0)
     await client2.sync();
+    assert.equal(
+      versionVectorHelper(doc2.getVersionVector(), [
+        { actor: client1.getID()!, lamport: BigInt(2) },
+        { actor: client2.getID()!, lamport: BigInt(3) },
+      ]),
+      true,
+    );
 
     doc2.update((root) => {
       delete root['2'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)